venv_update.py

```diff
@@ -328,7 +330,9 @@ def trace_requirements(requirements):
 
         for dist_req in sorted(dist.requires(), key=lambda req: req.key):
             # there really shouldn't be any circular dependencies...
```

Comment:
Ticket #52 was what I was anticipating with this comment.
The comment can go away now.